gdpr_assist/models.py

```diff
@@ -196,15 +200,12 @@ def get_export_filename(self):
             self.model._meta.object_name,
         )
 
+
 class PrivacyModel(models.Model):
     """
     An abstract model base class with support for anonymising data
     """
     anonymised = models.BooleanField(default=False)
-    retention_policy = models.ForeignKey(
```

Comment:
where did this go?

Reply:
We said we would manually set it on the models that needs it instead of being a default on all the models inheriting `PrivacyModel`. In our case, it'll just be `Course` to start with.

Reply:
hmm, okay. that makes sense. i was thinking more like having a `PrivacyModelWithRetentionPolicy` class to inherit from that would be a subclass of this one. either way is fine though

Reply:
Inheriting for just a field seems overkill... If we need generic methods to interact with `retention_policy` at the model level then we can reconsider.

Reply:
sounds good